src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -339,9 +354,43 @@ public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string cont
             }
         }
 
-        private static Expression<Func<ContentItemIndex, bool>> CreateOrderedContentIndexFilter(bool publishedOnly)
+        private IQuery<ContentItem> ContainedItemOptionsFilter(ContainedItemOptions containedItemOptions, IQuery<ContentItem> containedItems)
+        {
+            if (!string.IsNullOrEmpty(containedItemOptions.DisplayText))
+            {
+                containedItems.With<ContentItemIndex>().Where(i => i.DisplayText.Contains(containedItemOptions.DisplayText));
+            }
+
+            if (containedItemOptions.Status != ContentsStatus.None)
+            {
+                switch (containedItemOptions.Status)
+                {
+                    case ContentsStatus.Draft:
+                        containedItems?.With<ContentItemIndex>().Where(i => !i.Published);
+                        break;
+                    case ContentsStatus.Published:
+                        containedItems?.With<ContentItemIndex>().Where(i => i.Published);
+                        break;
+                    case ContentsStatus.Owner:
+                        string currentUserName = _hca.HttpContext?.User?.Identity?.Name;
```

Comment:
This has changed - in the last couple of days ;) to the `UserId`

So you have to access `NameIdentifier` to get the `currentUserId`
i.e. `HttpContext.User.FindFirstValue(ClaimTypes.NameIdentifier)`

Reply:
I see. I'm rewriting the code to what's suggested.